core/tools/generate_names.cc

```diff
@@ -180,6 +180,18 @@ NameDef names[] = {
     {"prefix"},
     {"to"},
 
+    {"mattr_accessor"},
+    {"mattr_reader"},
+    {"mattr_writer"},
+    {"cattr_accessor"},
+    {"cattr_reader"},
+    {"cattr_writer"},
+    {"instance_reader"},
+    {"instance_writer"},
+    {"instance_accessor"},
+    {"instance_predicate"},
+    {"class_attribute"},
```

Comment:
Can you add a second entry to these things that makes their C++ analogues use `camelCase`?

```
...
{"class_attribute", "classAttribute"}
...
```

(there should be more examples in the file to cargo cult).